andela/ah-backend-stark
# #160918236 clean up develop

#### What does this PR do?
reverts the settings file back to the way it was originally
#### Description of Task to be completed?
reverts the settings back to a single file rather than a folder with different versions of settings for the development and production environments
#### How should this be manually tested?
After cloning the repo and cd into it
 - you may check out the file structure in the authors app
 - when running the app locally, django-heroku MUST NOT be activated. To achieve this, lines 160 - 162 in authors/settings.py must be commented out, otherwise, the app will not run on localhost
#### Any background context you want to provide?
 - change debug to true in authors/settings.py for easy debugging, when running it locally
#### What are the relevant pivotal tracker stories?
#160918236
